# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1505498790 25200
#      Fri Sep 15 11:06:30 2017 -0700
# Node ID 230613ece415afc6fab31eead8a4cd68a1bedacb
# Parent  a666e433aa21171836d3116981885ce725a43c2b
Add the working directory to IMAGE_PATH when running in build tree

* run-octave.in (IMAGEPATH): Include "." in the default IMAGE_PATH to
match Octave's default behavior after installation.

diff --git a/run-octave.in b/run-octave.in
--- a/run-octave.in
+++ b/run-octave.in
@@ -44,17 +44,17 @@ d4_list=`$FIND "$d4" -type d -exec echo 
 d1_path=`echo "$d1_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 d2_path=`echo "$d2_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 d3_path=`echo "$d3_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 d4_path=`echo "$d4_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 
 octave_executable="$builddir/src/octave"
 
 LOADPATH="$d1_path:$d2_path:$d3_path:$d4_path"
-IMAGEPATH="$top_srcdir/scripts/image"
+IMAGEPATH=".:$top_srcdir/scripts/image"
 DOCFILE="$builddir/doc/interpreter/doc-cache"
 BUILT_IN_DOCSTRINGS_FILE="$builddir/libinterp/DOCSTRINGS"
 TEXIMACROSFILE="$top_srcdir/doc/interpreter/macros.texi"
 INFOFILE="$top_srcdir/doc/interpreter/octave.info"
 
 ## Checking for string equality below with prepended x's in order to
 ## handle problems with empty strings.
 while [ $# -gt 0 ]; do
